src/components/Details/styles.js

```diff
@@ -10,6 +10,15 @@ const styles = StyleSheet.create({
   movieCard: {
     height: 300,
   },
+  movieCardPlaceholder: {
+    backgroundColor: Colors.lightGray,
+    alignItems: 'center',
+    width: '100%',
```

Comment:
Is it necessary to have this and flex 1?

Reply:
none of them is necessary actually! removing...